src/DbUpdateService.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Microsoft.EntityFrameworkCore;
+using ModelRepoBrowser.Crawler;
+
+namespace ModelRepoBrowser;
+
+/// <summary>
+/// Service that periodically updates the Model Repository Database using <see cref="IRepositoryCrawler"/>.
+/// </summary>
+public class DbUpdateService : BackgroundService
+{
+    private readonly IServiceScopeFactory scopeFactory;
+    private readonly ILogger<DbUpdateService> logger;
+    private readonly IConfiguration configuration;
+
+    public TimeOnly PreferedTime { get; set; }
```

Comment:
REQ: Typo in Variable und entsprechenden Methoden fixen:
```suggestion
    public TimeOnly PreferredTime { get; set; }
```